plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/BridgeVifDriver.java

```diff
@@ -207,7 +207,7 @@ public LibvirtVMDef.InterfaceDef plug(NicTO nic, String guestOsType, String nicA
         String trafficLabel = nic.getName();
         Integer networkRateKBps = 0;
         if (libvirtVersion > ((10 * 1000 + 10))) {
-            networkRateKBps = (nic.getNetworkRateMbps() != null && nic.getNetworkRateMbps().intValue() != -1) ? nic.getNetworkRateMbps().intValue() * 128 : 0;
+            networkRateKBps = (nic.getNetworkRateMbps() != null && nic.getNetworkRateMbps().intValue() != -1) ? nic.getNetworkRateMbps().intValue() * 125 : 0;
```

Comment:
sorry to be that guy @Rubueno , but can you please replace "125" by a constant with a clear descriptive name? tnx

Reply:
Of course! Pushed :)

Reply:
@Rubueno I think that @DaanHoogland means a "private static int" at the top of the class. In theory this integer now can be modified and result in odd outcomes.